src/retasc/models/prerequisites/jira_issue.py

```diff
@@ -0,0 +1,181 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import os
+from collections.abc import Iterator
+from itertools import takewhile
+
+import yaml
+from pydantic import BaseModel, Field
+
+from retasc.jira import JIRA_ISSUE_ID_LABEL_PREFIX, JIRA_LABEL
+from retasc.models.release_rule_state import ReleaseRuleState
+from retasc.utils import to_comma_separated
+
+from .base import PrerequisiteBase
+
+
+def _is_resolved(issue: dict) -> bool:
+    return issue["fields"]["resolution"] is not None
+
+
+def _set_parent_issue(fields: dict, parent_issue_key: str | None = None):
+    if parent_issue_key is not None:
+        fields["parent"] = {"key": parent_issue_key}
+
+
+def _edit_issue(issue, fields, context, parent_issue_key: str | None = None):
```

Comment:
why not have this similar to _create_issue methond to `return issue` as well?

Reply:
There is no need for the return value.